### ISSUE TITLE
How do I run this?

### ISSUE THREAD

**Issue by MetroidSky** [encounter/retrotool#2]
I have downloaded the source code, how do I run this from the cmd?

**Comment by Istador** [comment]
You have to build a binary executable file from the source code that can then be run.

It's written in Rust, so you'd need to install that in order to build its source code.

Have a look at https://github.com/PrimeDecomp/retrotool/blob/main/.github/workflows/build.yml to see how it automatically builds itself on every new commit.

You can also download these automatic builded binaries here (select a workflow run and at the bottom there are artifacts):
https://github.com/PrimeDecomp/retrotool/actions

---

Basically if Rust is already installed:
```
# Windows amd64:
cargo build --release --all-features --target x86_64-pc-windows-msvc --bin retrotool

# Linux amd64:
pip install cargo-zigbuild
cargo zigbuild --release --all-features --target x86_64-unknown-linux-musl --bin retrotool
```
